### PR TITLE
[view] fix deadlock when creating airQualitySensor

### PR DESCRIPTION
**Pull request by michelececcacci**

### CODE REVIEW

**Comment by michelececcacci** [comment]
@smagnaniuni  this fixes everything. I will also probably create a logger that doesn't log temperature changes, since at the moment they basically make up for most of the logs and make debugging more difficult

**Comment by michelececcacci** [comment]
@smagnaniuni i added the type i needed in `AbstractActuatedDevice` and its state. should be good, since we talked about this today, but i'm pinging to warn about the change

File: src/main/java/homer/api/state/ActuatedDeviceState.java [hunk 1]
```diff
@@ -12,27 +12,44 @@ public final class ActuatedDeviceState implements DeviceState {
...
+     */
+    public static final String WINDOW = "Mechanized Window";
+    private final Optional<String> type;
 
     /**
      * Creates a new {@link ActuatedDeviceState} given the position and the bounds.
      * 
      * @param position       the actuator position.
      * @param positionBounds the actuator bounds.
+     * @param type
```
Comment (smagnaniuni) [inline src/main/java/homer/api/state/ActuatedDeviceState.java:34]: fix

File: src/main/java/homer/model/actuator/AbstractActuatedDevice.java [hunk 2]
```diff
@@ -12,20 +13,23 @@
...
 
     /**
      * Creates a {@link AbstractActuatedDevice} whose position is controlled by an
      * {@link Actuator}.
      * 
      * @param actuator The {@link Actuator} controlling the device.
+     * @param type
      */
-    public AbstractActuatedDevice(final Actuator actuator) {
+    public AbstractActuatedDevice(final Actuator actuator, final Optional<String> type) {
```
Comment (smagnaniuni) [inline src/main/java/homer/model/actuator/AbstractActuatedDevice.java:25]: Optionals in arguments make no sense, but whatever.

Comment (michelececcacci) [reply src/main/java/homer/model/actuator/AbstractActuatedDevice.java:25]: will fix

File: src/main/java/homer/api/state/ActuatedDeviceState.java (hunk 1 above)
Comment (michelececcacci) [reply src/main/java/homer/api/state/ActuatedDeviceState.java:34]: will fix

**Pull request merged**